web/src/i18n/en/index.ts

```diff
@@ -543,6 +543,17 @@ const en: BaseTranslation = {
       messages: {
         deleteApp: 'App and all tokens deleted.',
       },
+      warningModals: {
+        title: 'Warning',
+        content: {
+          usernameChange: `Changing a user's username has a significant impact on services they have logged into using Defguard. After changing the username, the user may lose access to applications (since they will not recognize them). Are you sure you want to proceed?`,
```

Comment:
```suggestion
          usernameChange: `Changing a username has a significant impact on services they have logged into using Defguard. After changing the username, a user may lose access to applications (since they will not recognize them). Are you sure you want to proceed?`,
```

Reply:
If you accept those changes, please generate again i18n-type.ts